lib/tasks/schedule.rake

```diff
@@ -119,7 +119,7 @@ task({ :reminder_rsvp=> :environment}) do
                  subject: 'Event Reminder From Movie Tsar',
                  html_body: "Just a friendly reminder that you have a party coming #{event.watch_date.strftime("%A %b %e")} at
                  #{event.watch_date.strftime("%l:%M %p")}.  The patry takes place at #{event.group.address}
-                 Please follow this link to confirm your rsvp",
+                 Please follow this link to confirm your rsvp", 
```

Comment:
I assume a link will be put in the email eventually. A one-time use link that updates the user's RSVP to `"accepted"`